src/panthor_drm.h

```diff
@@ -0,0 +1,876 @@
+/* SPDX-License-Identifier: MIT */
```

Comment:
Shouldn't this header part of the Linux headers?
I guess that if it's too recent we can keep the header until it's widely available if it's stable enough

Reply:
My understanding is that this sort of uAPI file isn't part of the usual kernel headers package under /usr/include/linux. Both Mes and the IGT test suit have to keep the same uAPI files up to date with the one in the kernel sources, so I thought we'd have to do the same for nvtop.